src/styles/stars.scss

```diff
@@ -0,0 +1,23 @@
+@import 'variables';
+
+.stars {
+  display: flex;
+  list-style: none;
+
+  &__star {
+    width: $star-size;
+    height: $star-size;
+    margin-top: 16px;
+    margin-right: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+  }
+
+  &__star--1:nth-child(-n + 1),
```

Comment:
In this case you can use scss loop.
Check it and try.  
Also you need 5 stars variant ( not only 1-4 that you made) .
Thanks)